packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -566,6 +571,12 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
         this.addKeyListener(this.node, up, event => this.handleUp(event));
         this.addKeyListener(this.node, down, event => this.handleDown(event));
         this.addKeyListener(this.node, Key.ENTER, event => this.handleEnter(event));
+        this.node.addEventListener('ps-scroll-y', (e: any) => {
```

Comment:
`e: { target: { scrollTop: number } }`

Reply:
That does not work. The compiler complains.
```
[typescript]
Argument of type '(e: { target: { scrollTop: number; }; }) => void' is not assignable to parameter of type 'EventListenerOrEventListenerObject'.
  Type '(e: { target: { scrollTop: number; }; }) => void' is not assignable to type 'EventListenerObject'.
    Property 'handleEvent' is missing in type '(e: { target: { scrollTop: number; }; }) => void'.
```

Reply:
I assume this listener should be removed later, right?